webpack/envs/clientDevelopmentConfig.js

```diff
@@ -26,6 +26,7 @@ export const clientDevelopmentConfig = {
   devtool: env.webpackDevtool || "eval",
   stats: env.webpackStats || "errors-only",
   module: {
+    // Why do we only compile css in development mode?
```

Comment:
CSS is compiled and fingerprinted external to webpack, when in prod. This was migrated over from old browserify setup, but not sure why: https://github.com/artsy/force/blob/81eb937dcc05b112f647c09fe2e4cc7051453d05/webpack/utils/getCSSManifest.js#L1

And then here is where its called: https://github.com/artsy/force/blob/81eb937dcc05b112f647c09fe2e4cc7051453d05/webpack/envs/productionConfig.js#L33